lib/util/EscapeUtil.js

```diff
@@ -2,13 +2,20 @@ export {
   default as escapeCSS
 } from 'css.escape';
 
+var UNSAFE_HTML = /[&<>"']/g;
+
 var HTML_ESCAPE_MAP = {
-  '<': '&lt',
-  '>': '&gt'
+  '&': '&amp;',
+  '<': '&lt;',
+  '>': '&gt;',
+  '"': '&quot;',
+  '\'': '&#39;'
 };
 
 export function escapeHTML(str) {
-  return str.replace(/[<>]/g, function(match) {
+  str = '' + str;
```

Comment:
Regexes are stateful per default. Can we inline the `UNSAFE_HTML` tag?

Reply:
Why are we escaping the extra characters, too? What is the purpose?

Reply:
> Regexes are stateful per default. Can we inline the `UNSAFE_HTML` tag?

Sure, I'll do this.

> Why are we escaping the extra characters, too? What is the purpose?

If unescaped, ampersand will swallow a few following characters. 

Regarding the quotation marks, they might be a problem if inserted as part of attribute. However, I haven't found any example of this in the libraries. I didn't look for it in all of them, though.

Reply:
All right. Let's go for full escaping then and see what happens.